vss-cats/src/main/protobuf/password.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
```

Comment:
If there's an issue for it I'll change it in a separate PR, there is currently a chain of 3 PRs depending on this one and I want to avoid unnecessary conflicts.

Reply:
I can take it when all required PRs are merged.